lib/diagnostic.js

```diff
@@ -5,17 +5,23 @@
 // the minimum value in an array
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let minimum = array.reduce(min);`
-const min = function (/* previous, current */) {
-
+const min = function (previous, current, curIndex, a) {
```

Comment:
FYI, the `(/* previous, current */)` in the boilerplate code is supposed to be a hint about what arguments are being passed into this function